schemas/property-units.yaml

```diff
@@ -76,12 +76,18 @@ patternProperties:
   "-micro-ohms$":
     $ref: "types.yaml#/definitions/uint32-array"
     description: microohm
+  "-microwatt$":
+    $ref: "types.yaml#/definitions/uint32-array"
+    description: microwatt
   "-microwatt-hours$":
     $ref: "types.yaml#/definitions/uint32-array"
     description: microwatt hour
   "-microvolt$":
     $ref: "types.yaml#/definitions/uint32-matrix"
     description: microvolt
+  "-milliwatt$":
```

Comment:
Group same units together. I guess after microvolts would be best to have some level of alphabetical ordering of the base unit.